graphql/executor.go

```diff
@@ -175,6 +197,11 @@ func (e *Executor) resolveAndExecute(ctx context.Context, field *Field, source i
 		}), nil
 	}
 
+	defer func() {
```

Comment:
i think it might make sense to only trace the expensive resolvers here, since the non-expensive ones should (hopefully) take trivial amounts of time to compute.